app/src/main/java/org/ole/planet/myplanet/service/UploadManager.java

```diff
@@ -121,6 +124,49 @@ public void uploadAchievement() {
     }
 
 
+    public void uploadSubmitPhotos(SuccessListener listener)
```

Comment:
@jeevast you just changed the name , just use one method for uploading attachment, into any db.
eg : 
```
public void uploadAttchment(String tableName, String path, ...Other params.....){
//upload task here
}
```